pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -336,6 +340,13 @@ def on_validation_end(self, trainer, pl_module):
         metrics = trainer.logger_connector.callback_metrics
         epoch = trainer.current_epoch
 
+        # backward compatibility... need to deprecate
```

Comment:
these need to check if `self.monitor is None` before resetting it to the older defaults right?

Reply:
yup! sorry, i think i removed the WIP too soon haha. but good catch

Reply:
ended up just doing a much needed clean up to maintain sanity haha